.github/workflows/ci.yml

```diff
@@ -0,0 +1,26 @@
+name: Build
+
+on:
+  pull_request:
+  push:
+    branches: ['**']
+    tags: [v*]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Build and Push Docker Image
+        uses: Dwolla/jenkins-agents-workflow/.github/actions/build@main
+        with:
+          DOCKERHUB_USERNAME: ${{ secrets.DOCKERHUB_USERNAME }}
+          DOCKERHUB_TOKEN: ${{ secrets.DOCKERHUB_TOKEN }}
+          BASE_TAG: <my-specified-base-image-tag>
+          TAG_NAME: <my-unsupported-tag-name>
```

Comment:
@gorskilauren is there a way to derive this from the value in the Dockerfile? So as of this PR, it would be `v0.35.0`.

I'm guessing that the available tooling to keep docker images up to date works on the tags in the Dockerfile, not elsewhere, so I was thinking it would be nice not to have the value come from this workflow file. Open to suggestions though.

Reply:
> I'm guessing that the available tooling to keep docker images up to date works on the tags in the Dockerfile

Are you thinking for tooling that scans docker images in source?

Reply:
Yep, exactly. [Here's a blog](https://dev.to/oracle2025/how-to-keep-a-dockerfile-updated-with-dependabot-1mdn) describing how to do it with Dependabot, for example.